### DISCUSSION TITLE
umi使用React.lazy来生成路由，如果想替代为react-loadable的方式呢？

### DISCUSSION THREAD

**Discussion by Lemonreds** [umijs/umi#11835] [Q&A] [answered]
想使用react-loadable的预加载chunk功能，这里是没有办法修改了吗？

代码片段见：
https://github.com/umijs/umi/blob/a8303837e2f33c8a82642b302ac57cf367af1bee/packages/preset-umi/src/features/tmpFiles/routes.ts

**Comment by fz6m** [comment, marked as answer]
在 umi 4 中，放弃了对旧浏览器、IE 的兼容，默认兼容到现代浏览器，故不再使用 loadable ，目前无法修改。
